src/__experimental__/components/date-range/date-range.spec.js

```diff
@@ -171,163 +175,146 @@ describe('DateRange', () => {
+      endInput = wrapper.find(DateInput).at(1).childAt(0).childAt(0);
+      expect(startInput.props().disabled).toBeUndefined();
+      expect(endInput.props().disabled).toBeUndefined();
     });
 
     it('dates can be disabled by passing startDateProps and endDateProps to DateRange', () => {
-      const labelInstance = TestUtils.renderIntoDocument(
-        <DateRange
-          onChange={ customOnChange }
-          value={ ['2016-10-10', '2016-11-11'] }
-          startDateProps={ { disabled: true } }
-          endDateProps={ { disabled: true } }
-        />
-      );
-      const dates = TestUtils.scryRenderedComponentsWithType(labelInstance, Date);
-      expect(dates[0].props.disabled).toEqual(true);
-      expect(dates[1].props.disabled).toEqual(true);
+      wrapper = renderDateRange({
+        onChange: customOnChange,
+        startDateProps: { disabled: true },
+        endDateProps: { disabled: true },
+        value: []
+      }, mount);
+      startInput = wrapper.find(DateInput).at(0).childAt(0).childAt(0);
+      endInput = wrapper.find(DateInput).at(1).childAt(0).childAt(0);
+      expect(startInput.props().disabled).toEqual(true);
+      expect(endInput.props().disabled).toEqual(true);
     });
 
     it('Date values can be set via startDateProps and endDateProps', () => {
-      const labelInstance = TestUtils.renderIntoDocument(
-        <DateRange
-          onChange={ customOnChange }
-          startDateProps={ { value: '2016-10-10' } }
-          endDateProps={ { value: '2016-11-11' } }
-          value={ [] }
-        />
-      );
-
-      const dates = TestUtils.scryRenderedComponentsWithType(labelInstance, Date);
-      expect(dates[0].props.value).toEqual('2016-10-10');
-      expect(dates[1].props.value).toEqual('2016-11-11');
+      wrapper = renderDateRange({
+        onChange: customOnChange,
+        startDateProps: { value: '2016-10-10' },
+        endDateProps: { value: '2016-11-11' },
+        value: []
+      }, mount);
+      startInput = wrapper.find(DateInput).at(0).childAt(0).childAt(0);
+      endInput = wrapper.find(DateInput).at(1).childAt(0).childAt(0);
+      expect(startInput.props().value).toEqual('2016-10-10');
+      expect(endInput.props().value).toEqual('2016-11-11');
     });
 
     it('value prop is retained for backward compatibility', () => {
-      const labelInstance = TestUtils.renderIntoDocument(
-        <DateRange
-          onChange={ customOnChange }
-          value={ ['2015-10-10', '2015-11-11'] }
-        />
-      );
-      const dates = TestUtils.scryRenderedComponentsWithType(labelInstance, Date);
-      expect(dates[0].props.value).toEqual('2015-10-10');
-      expect(dates[1].props.value).toEqual('2015-11-11');
+      wrapper = renderDateRange({
+        onChange: customOnChange,
+        value: ['2015-10-10', '2015-11-11']
+      }, mount);
+      startInput = wrapper.find(DateInput).at(0).childAt(0).childAt(0);
```

Comment:
I think these `childAt()` tests are quite a brittle, if the internal composition of the Date Input changes these tests will fail, could we not do something like `wrapper.find(DateInput).at(0).find(ComponentName)`?

Reply:
fixed.